docs/tutorial/procedure.rst

```diff
@@ -308,49 +317,52 @@ Now that you have a background on how to use the different features of the Proce
 
     # Import necessary packages
     from pymeasure.instruments.keithley import Keithley2400
-    from pymeasure.experiment import Procedure
+    from pymeasure.experiment import Procedure, Results, Worker
     from pymeasure.experiment import IntegerParameter, FloatParameter
     from time import sleep
+    import numpy as np
+
+    from pymeasure.log import log, console_log
 
     class IVProcedure(Procedure):
 
-        data_points = IntegerParameter('Data points', default=50)
-        averages = IntegerParameter('Averages', default=50)
-        max_current = FloatParameter('Maximum Current', units='A', default=0.01)
-        min_current = FloatParameter('Minimum Current', units='A', default=-0.01)
+        data_points = IntegerParameter('Data points', default=20)
+        averages = IntegerParameter('Averages', default=8)
+        max_current = FloatParameter('Maximum Current', units='A', default=0.001)
+        min_current = FloatParameter('Minimum Current', units='A', default=-0.001)
 
         DATA_COLUMNS = ['Current (A)', 'Voltage (V)', 'Voltage Std (V)']
 
         def startup(self):
             log.info("Connecting and configuring the instrument")
-            self.sourcemeter = Keithley2400("GPIB::4")
+            self.sourcemeter = Keithley2400("GPIB::24", includeSCPI=True)
```

Comment:
```suggestion
            self.sourcemeter = Keithley2400("GPIB::24")
```
Either an instrument supports SCPI or it does not. Therefore it should be called only in an instrument implementation, not when you create an instrument.

Reply:
Ok. Got. Thanks